docs/docsite/rst/dev_guide/testing/sanity/action-plugin-docs.rst

```diff
@@ -0,0 +1,4 @@
+Sanity Tests » action-plugin-docs
```

Comment:
Needs the `:orphan:` marker, or the docs build will break.

Reply:
The tests pass without adding `:orphan:` and most of the sanity docs pages do not have it. We shouldn't add it without verifying it's required. If it is required, we should figure out why the test passes.

Reply:
Huh, weird - I wonder why some require it and some don't? Well, I'm okay with merging this as is, then. And I'll explore the question independently.